bin/plugin/commands/performance.js

```diff
@@ -214,24 +196,47 @@ async function runPerformanceTests( branches, options ) {
 	// 1- Preparing the tests directory.
 	log( '\n>> Preparing the tests directories' );
 	log( '    >> Cloning the repository' );
-	const baseDirectory = await git.clone( config.gitRepositoryURL );
+
+	/**
+	 * @type {string[]} git refs against which to run tests;
+	 *                  could be commit SHA, branch name, tag, etc...
+	 */
+	if ( branches.length < 2 ) {
+		throw new Error( `Need at least two git refs to run` );
+	}
+
+	const baseDirectory = getRandomTemporaryPath();
+	fs.mkdirSync( baseDirectory, { recursive: true } );
+
+	// @ts-ignore
+	const git = SimpleGit( baseDirectory );
+	await git
+		.raw( 'init' )
+		.raw( 'remote', 'add', 'origin', config.gitRepositoryURL );
```

Comment:
Could use `await` [`git.init()`](https://github.com/steveukx/git-js#git-init)[`.addRemote( 'origin', config.gitRepositoryURL )`](https://github.com/steveukx/git-js#git-remote), but I guess you've covered that [below](https://github.com/WordPress/gutenberg/pull/45255/files#r1008373526).

Reply:
thanks. I tried changing it but there's a thing I don't like about this pattern which is that `SimpleGit` is making us guess which functions it has chosen to abstract and which not to, and then we need to guess what that abstraction does.

case in point, we have `SimpleGit.addRemote()` but not `SimpleGit.showRemote()`. So while `.init()` makes sense, I feel like the raw version of `remote add` does far less to obscure what's going on, even if in the end it does the same thing as raw does.

I'd like to leave this as is for now until and unless we see a compelling reason why using this middleman/new term is more useful than using the more true `git` vocabulary and idioms.